includes/Helpers/LogHelper.php

```diff
@@ -139,6 +139,7 @@ public static function getLogDescription(Log $entry)
         $lookup = array(
             'Reserved'            => 'reserved',
             'Email Confirmed'     => 'email-confirmed',
+            'Manually Confirmed'     => 'manually confirmed the request',
```

Comment:
We should probably add this to `getLogActions(...)` below so this log type shows up on the filters on the log page.